config.json

```diff
@@ -70,7 +76,7 @@
         "port": 9991
     },
     "recordLog": {
-        "enabled": false,
+        "enabled": true,
```

Comment:
No sure it is useful for people using DMD. And Mongo does not use this param.

Reply:
it works explicitly for dmd